pylib/zeus/zeus.py

```diff
@@ -30,15 +30,20 @@ def find_zeus_executable(current_path):
   :return: Path to the executable or None if not found.
   """
 
+  wanted_steps = (None, None)
```

Comment:
I don't understand why we use this tuple.
e.g. - if we start in blah/generate/run - then 1st iteration we start looking for zeus_run, 2nd iteration we look for zeus_run again, in the generate folder? I don't understand what we're trying to catch here?
Don't we only care what the current folder name is?

Reply:
Unfortunately no, it's a workaround for the mess we made:
```
pipeline/
  vtap/
    process/
      run/
      update/
      zeus_process
      zeus_run
```
you see how postfix for the script is defined by directory name at different level